core/src/com/unciv/ui/cityscreen/CityStatsTable.kt

```diff
@@ -74,17 +74,26 @@ class CityStatsTable(val cityScreen: CityScreen): Table() {
                     -> "Food converts to production"
                     else -> "Stopped population growth"
                 }.tr()
-        turnsToPopString += " (${cityInfo.population.foodStored}/${cityInfo.population.getFoodToNextPopulation()})"
+        turnsToPopString += " (${cityInfo.population.foodStored}${Fonts.food}/${cityInfo.population.getFoodToNextPopulation()}${Fonts.food})"
 
         innerTable.add(unassignedPopString.toLabel()).row()
         innerTable.add(turnsToExpansionString.toLabel()).row()
         innerTable.add(turnsToPopString.toLabel()).row()
-        if (cityInfo.isInResistance())
-            innerTable.add("In resistance for another [${cityInfo.getFlag(CityFlags.Resistance)}] turns".toLabel()).row()
-        if (cityInfo.isWeLoveTheKingDay())
-            innerTable.add("We Love The King Day for another [${cityInfo.getFlag(CityFlags.WeLoveTheKing)}] turns".toLabel()).row()
-        else if (cityInfo.demandedResource != "")
-            innerTable.add("Demanding [${cityInfo.demandedResource}]".toLabel()).row()
+        
+        val tableWithIcons = Table()
+        tableWithIcons.defaults().pad(2f)
+        if (cityInfo.isInResistance()) {
+            tableWithIcons.add(ImageGetter.getImage("StatIcons/Resistance")).size(20f)
+            tableWithIcons.add("In resistance for another [${cityInfo.getFlag(CityFlags.Resistance)}] turns".toLabel()).row()
+        }
+        if (cityInfo.isWeLoveTheKingDay()) {
+            tableWithIcons.add(ImageGetter.getStatIcon("Food")).size(20f)
```

Comment:
Nitpicky/highly subjective: Maybe the `Population` stat icon?

Reply:
Was my first idea as well, but we don't have that as an emoji yet

Reply:
To me food seems more logical, as the main benefit is the +20% Growth (surplus food) it gives, but an argument could be made for both